### ISSUE TITLE
LS to NS - Credit Memo

### ISSUE THREAD

**Issue by dj-tkoizumi** [dj-tkoizumi/boomi#2]
Hi Takayuki,
 
This is an error coming from Netsuite when the integration tries to insert the Credit Memo.   This error has been occurring from months.  I’ve already informed Amanda and Melissa to find out what field Netsuite Credit Memo is requiring.     Previously, Credit Memos went through fine.  Something changed in Netsuite Credit Memo logic that must’ve have added a new field requirement.   If someone can explain what is missing, then I can update the integration.
 
I also suggested for someone to enter the Credit Memo manually into Netsuite to see if the same error occurs from the front end (it might provide more details as to what field is required).
 
 
Regards,
Jeff

**Comment by dj-tkoizumi** [comment]
Customer
--
Date
Posting Period
Order Type

**Comment by dj-tkoizumi** [comment]
Hi Takayuki,
 
The integration populates all of these except Posting Period.    Posting Period is a field that is dynamically determined by Netsuite when inserting a Credit Memo.
 
However, while researching, I discovered that the integration is actually inserting the Credit Memos into Netsuite.   For example, on the attached email error notification, it shows an error for LS # 89939.
 
However, when I search in Netsuite for Credit Memos and include the EXTERNAL_ID field (which the integration inserts into NS), I see the following:
 

 
 
So then, I clicked on “Edit” for Credit Memo 1803420 (LSR_89939), and clicked on “Save” (without making any change to see if it gave any issues).    And then I got this error page:
So basically, there is a SuiteScript in Netsuite that is causing a problem.  Someone with Netsuite experience needs to research this script issue.
